docs/plugin_development.rst

```diff
@@ -41,167 +52,208 @@ To quickly get started in desktop application development with Qt and Python her
+Spyder was developed in Qt, and requires for its operation two packages with which it is closely related (and without which it cannot work): `Spyder-Kernels`_ and `Python-LSP-Server`_.
 
-* Spyder uses `qtpy`_ which is an abstraction layer that allows you to work with Qt from Python regardless of whether you use either of the two reference libraries: PyQt or PySide.
+* `Qt`_ is an open source multiplatform widget toolkit for creating native graphical user interfaces.
+  Qt is a very complete development framework that offers utilities for building applications and has extensions for networking, Bluetooth, charts, 3D rendering, navigation (such as GPS), among others.
 
-* `spyder-kernels`_ provide Jupyter kernels to Spyder, for use within its consoles.
+* Spyder uses `QtPy`_ which is an abstraction layer that allows you to work with Qt from Python regardless of whether you use one of the two reference libraries: PyQt or PySide.
 
 .. important::
 
-   Spyder is currently developed in such a way that most of its features are implemented as plugins.
+   Spyder is currently being developed in such a way that most of its features are implemented as plugins.
+
+If you are interested in learning more about Spyder's features, you can read about the project on `Spyder`_'s website.
+If you are interested in contributing to Spyder, please visit the `contribution guide`_.
 
 .. _contribution guide: https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md
 .. _Qt: https://www.qt.io/
-.. _qtpy: https://github.com/spyder-ide/qtpy
-.. _spyder-kernels: https://github.com/spyder-ide/spyder-kernels
-.. _pyls-spyder: https://github.com/spyder-ide/pyls-spyder
-.. _python-lsp-server: https://github.com/python-lsp/python-lsp-server
+.. _QtPy: https://github.com/spyder-ide/qtpy
+.. _Spyder-Kernels: https://github.com/spyder-ide/spyder-kernels
+.. _Python-LSP-Server: https://github.com/python-lsp/python-lsp-server
 
 
 
-=========================================
-Types of plugins we can develop in Spyder
-=========================================
+.. _tutorial-plugin-concepts:
 
+=========================
+Developing Spyder plugins
+=========================
 
-.. image:: images/workshop-3/pd_spyder_plugins.png
-   :alt: Types of Spyder plugins.
+Now that we know a little bit about the basics of Spyder, it's time to dive into the heart of this tutorial: the plugins that can be developed in Spyder.
 
-.. note::
+.. image:: images/workshop-3/pd_spyder_plugins.png
+   :alt: Types of Spyder plugins
 
-    A plugin is a component that adds functionality to an application, it can be a graphical component, for example, to display maps, or a non-graphical one that adds additional syntax coloring schemes.
+First, let's define what a plugin is.
+*A plugin is a component that adds functionality to an application; it can be a graphical component, for example, to display maps, or a non-graphical one that adds additional syntax coloring schemes.*
```

Comment:
@ccordoba12

Reply:
As far as I can tell, despite appearing as a quote it is actually original to this document, as this page is itself the top hit on Google, DDG and Startpage when I searched for the sentence, with nothing else even that close.

Therefore, to avoid the misunderstanding in the first place, I suggest the straightforward syntax change:

```suggestion
A *plugin* is a component that adds functionality to an application; it can be a graphical component, for example, to display maps, or a non-graphical one that adds additional syntax highlighting schemes.
```